infra-templates/vxlan/10/rancher-compose.yml

```diff
@@ -2,6 +2,7 @@
   name: "Rancher VXLAN"
   version: "v0.1.2"
```

Comment:
we should bump the version to v0.2.0, if these are fresh install only. you'll need to also bump up the version in the rancher-compose.yml